src/blog/templates/blog_search_tag.html

```diff
@@ -0,0 +1,97 @@
+{% extends "blog_base.html" %}
+{% load static %}
+
+{% block metatags %}
+    <meta name="description" content="{% block metadesc %}Selfpower.ro - Blog, Articole, Studii, Index{% endblock metadesc %}">
+    <meta name="keywords" content="Selfpower, Blog, Articole, Studii, Consiliere, Educatie, Dezvoltare, Personala">
+    <meta property="og:title" content="{% block metatitle %}Selfpower Blog - Articole despre Consiliere, Educatie si Dezvoltare Personala{% endblock metatitle %}" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.selfpower.ro/blog/{% block og_url %}{% endblock og_url %}" />
+    <meta property="og:description" content="Selfpower - Blog, Articole, Studii" />
+    <meta property="og:image" content="https://www.selfpower.ro/static/img/banner.png" />
```

Comment:
Aici informativ, cu meta og:image si urmatoarele specifici ce imagine vrei sa apara pe FB cand faci share la pagina de html in care apar aceste meta tag-uri.
FB are nevoie de dimensiunea imaginii, tip-ul imaginii, daca nu le specifici FB nu stie ce sa arate in preview cand faci share si uneori ai surprize sa iti arate o imagine aleatoare sau prost incadrata.